configure

```diff
@@ -6685,6 +6689,13 @@ fi
 define_in_config_if_yes "$_discord" 'USE_DISCORD'
 echo "$_discord"
 
+
+#
+# Check for Imgui
+#
```

Comment:
What is missing here is ```echocheck "ImGui". Currently, the execution looks like:
```
Checking for Discord RPC... no
yes
Building translation support... yes (with runtime language detection)
```